common/event_modifiers/31_pony.txt

```diff
@@ -17,6 +17,19 @@ running_of_the_leaves = {
 	production_efficiency = 0.10
 }
 
+magic_assimilation = {
+	culture_conversion_cost = -0.8
+	missionaries = 2
+	global_missionary_strength = 0.8
+}
```

Comment:
Same goes for the Pressence one. United Ponykind is just a nice subtle boost, it seemed fitting.